CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - The `--help` output was reworked with a new colorful syntax
 - OSC 52 is now disabled on unfocused windows
 - `SpawnNewInstance` no longer inherits initial `--command`
+- `--embed` could accept hexadecimal values starting with `0x`
```

Comment:
This is a new feature, should be moved to `Added`. "could" is also not appropriate here, I recommend the following:

```
 - Accept hexadecimal values starting with `0x` for `--embed`
```